src/main/java/org/zeromq/ZMQ.java

```diff
@@ -447,18 +509,23 @@ public static void sleep(long amount, TimeUnit unit)
 
         public static Error findByCode(int code)
         {
-            for (Error e : Error.values()) {
-                if (e.getCode() == code) {
-                    return e;
-                }
+            if (map.containsKey(code)) {
+                return map.get(code);
+            }
+            else {
+                throw new IllegalArgumentException("Unknown " + Error.class.getName() + " enum code:" + code);
```

Comment:
could use a space here between the colon and the code:

```suggestion
                throw new IllegalArgumentException("Unknown " + Error.class.getName() + " enum code: " + code);
```

Reply:
Indeed